test/org/opensolaris/opengrok/web/DirectoryListingTest.java

```diff
@@ -84,11 +85,15 @@ private void create() throws Exception {
             assertTrue("Failed to set modification time",
                        file.setLastModified(val));
 
-            if (size > 0) {
-                FileOutputStream out = new FileOutputStream(file);
-                byte[] buffer = new byte[size];
-                out.write(buffer);
-                out.close();
+            try {
+                int s = Integer.parseInt(size);
+                if (s > 0) {
+                    FileOutputStream out = new FileOutputStream(file);
+                    byte[] buffer = new byte[s];
+                    out.write(buffer);
+                    out.close();
+                }
+            } catch (NumberFormatException e) {
```

Comment:
I think the better way would be to explicitly check for `-`. If the string cannot be parsed as integer, then the test should fail. Also, it is generally considered bad practice to include empty catch statements (at least some explanatory test should be added).